.deny.toml

```diff
@@ -0,0 +1,6 @@
+[licenses]
+allow = [
+	"MPL-2.0",
```

Comment:
I would prefer just spelling out all licenses explicitly.

Reply:
Easy enough; I did it originally to get a feel for our dep tree, so it's in my undo stack. 😎

Just pushed new changes (destructively).